miner/scroll_worker.go

```diff
@@ -904,6 +910,11 @@ func (w *worker) commit(reorging bool) (common.Hash, error) {
 	}
 	cccStallTimer.UpdateSince(checkStart)
 
+	// Reset the pending transactions if we have reached the deadline or every 10th block
+	if w.current.deadlineReached || w.current.header.Number.Uint64()%10 == 0 {
```

Comment:
Why every 10th block? Even if we can manage to produce 10 blocks within 10s this is still a long time period. This will have a significant effect on tx ordering and people might bid higher and higher without being included in this timeframe.

Reply:
Discussed on slack.